stubs/Authlib/authlib/oauth2/rfc9068/introspection.pyi

```diff
@@ -0,0 +1,13 @@
+from _typeshed import Incomplete
+
+from authlib.oauth2.rfc7662 import IntrospectionEndpoint
+
+class JWTIntrospectionEndpoint(IntrospectionEndpoint):
+    ENDPOINT_NAME: str
+    issuer: Incomplete
+    def __init__(self, issuer, server: Incomplete | None = None, *args, **kwargs) -> None: ...
```

Comment:
`issuer, *, server: Incomplete | None = None, **kwargs`

The source function only has kwargs

https://github.com/lepture/authlib/blob/master/authlib/oauth2/rfc9068/claims.py#L17